pilot/pkg/proxy/envoy/watcher_test.go

```diff
@@ -124,14 +129,15 @@ func TestWatchCerts(t *testing.T) {
 
 	ctx, cancel := context.WithCancel(context.Background())
 
-	go watchCerts(ctx, []string{name}, watchFileEvents, 50*time.Millisecond, callbackFunc)
+	go watchCerts(ctx, []string{tmpFile.Name()}, watchFileEvents, 50*time.Millisecond, callbackFunc)
```

Comment:
Can we add a test that covers the original failure? I think we can add to the end of this test deleting the file and making sure the change is detected?

Reply:
Sure.